packages/extension/src/view/devtools/pages/learning/contentPanel.tsx

```diff
@@ -17,67 +17,24 @@
  * External dependencies.
  */
 import React from 'react';
-import {
-  DevGuideIcon,
-  Help,
-  SIDEBAR_ITEMS_KEYS,
-  useSidebar,
-  WebStoriesIcon,
-  WikiIcon,
-} from '@google-psat/design-system';
-
-const ITEMS = [
-  {
-    name: 'Wiki',
-    icon: WikiIcon,
-    sidebarKey: SIDEBAR_ITEMS_KEYS.WIKI,
-  },
-  {
-    name: 'Stories',
-    icon: WebStoriesIcon,
-    sidebarKey: SIDEBAR_ITEMS_KEYS.STORIES,
-  },
-  {
-    name: 'Help Center',
-    icon: Help,
-    sidebarKey: SIDEBAR_ITEMS_KEYS.HELP_CENTER,
-  },
-  {
-    name: 'Dev Site',
-    icon: DevGuideIcon,
-    sidebarKey: SIDEBAR_ITEMS_KEYS.DEV_SITE,
-  },
-];
+import { useSidebar } from '@google-psat/design-system';
+import CardsPanel from '../shared/cardsPanel';
+import { FEATURED_ITEMS } from './constants';
 
 const ContentPanel = () => {
   const navigateTo = useSidebar(({ actions }) => actions.updateSelectedItemKey);
 
-  return (
-    <div
-      data-testid="learning-content"
-      className="h-full w-full overflow-auto text-raisin-black dark:text-bright-gray px-2 pb-14"
-    >
-      <div className="min-w-[45.75rem]">
-        <section className="mb-5 pb-5">
-          <div className="flex gap-x-5 gap-y-4 flex-wrap">
-            {ITEMS.map((item) => {
-              const Icon = item.icon;
+  const onButtonClick = (event: React.MouseEvent, sidebarKey: string) => {
```

Comment:
Should the `sidebarKey` be using type `SIDEBAR_ITEMS_KEYS` ?

Reply:
Will it be better if we use `useCallback` so the function doesn't get defined on every render?